tasks/image-augmentation/main.py

```diff
@@ -49,7 +49,8 @@ def main() -> None:
         flipH = taskRun.parameters["flipHorizontalPrc"]
         affine = taskRun.parameters["affine"]
         noise = taskRun.parameters["noise"]
-        blur = taskRun.parameters["blurPercentage"]
+        blurPrc = taskRun.parameters["blurPercentage"]
```

Comment:
Maybe "blurPtc" has more sense for blurPercentage var name since Ptc is shortcut for percentage?

Reply:
Pct seems to be the most common abbreviation, so I'll go with that one